index_matroska.md

```diff
@@ -78,7 +78,7 @@ the [IETF Cellar working group](https://datatracker.ietf.org/wg/cellar/charter/)
 But since it's quite complete it is used as a reference for the development of libmatroska.
 A legacy version of the specification can be found [here](https://www.matroska.org/files/matroska_file_format_alexander_noe.pdf) (PDF doc by Alexander Noé -- outdated).
 
-For a simplified diagram of the layout of a Matroska file, see the [Diagram page](diagram.md).
+For a simplified diagram of the layout of a Matroska file; see the [Diagram page](diagram.md).
```

Comment:
Same argument as above: the "replace with full stop" test fails. A comma is appropriate, a semicolon isn't.